lib/helpers/UiBaseLocators.ts

```diff
@@ -132,33 +134,26 @@ export class UiBaseLocators {
     await this.page.waitForTimeout(1000);
     await this.modalCaretBtn.click({force: true});
     await this.page.getByLabel(dictionaryName).click();
-    await this.chooseDictionaryBtn.click();
-  }
-
-  // TODO: replace this method by addQueryBuilderWithOrderByStatement() in all tests and remove it
-  async addQueryBuilderWithCreateDateOption() {
-    await this.queryBuilderBtn.click({force: true});
-    await this.page.waitForTimeout(1000);
-    await this.queryBuilderOrderedBy.click({force: true});
-    await this.page.waitForTimeout(1000);
-    await this.queryBuilderCreateDate.click({force: true});
-    await this.submitBtn.click({force: true});
+    await this.chooseThreeDotsBtn.click();
   }
 
-  async addQueryBuilderWithOrderByStatement(propertyAlias: string, isAcsending: boolean) {
+  async addQueryBuilderWithOrderByStatement(propertyAlias: string, isAscending: boolean) {
     await this.queryBuilderBtn.click({force: true});
+    await this.page.waitForTimeout(500);
     // Wait and click to orderBy dropdownbox
     await expect(this.orderByPropertyAliasBtn).toBeVisible({timeout: 1000});
     await this.orderByPropertyAliasBtn.click({force: true});
     // Wait and choose property alias option 
     await this.waitAndSelectQueryBuilderDropDownList(propertyAlias);
     // Click to acending button if isAcsending is false
-    if (!isAcsending) {
-      await this.acsendingBtn.click({force: true});
+    if (!isAscending) {
+      await this.ascendingBtn.click({force: true});
     }
+    await this.submitBtn.click({force: true});
```

Comment:
We should not include this step in this method because we need to verify whether the code is displayed correctly or not before clicking the submit button